content/news/011.ripe-labs-article-competition.de.md

```diff
@@ -0,0 +1,18 @@
+---
+published: 2024-10-03
+description: IXP-from-Scratch Serie gewinnt RIPE Labs Artikel Wettbewerb
+title: RIPE Labs Artikel Wettbewerb gewonnen
+keywords: [ Event ]
+authors: [ Maurice ]
+image: !TODO
+---
+
+Am 12. August erschien unser erster Artikel auf RIPE Labs für die Serie 
+["IXP from Scratch"](https://labs.ripe.net/author/liske/ixp-from-scratch-building-a-new-ix/). Wir beschreiben in der Serie
+die wesentlichen Punkte zum Aufbau eines IXPs und welche Probleme und Anforderungen wir bisher überwunden haben. 
+
+Kurz darauf wurde der RIPE Labs Artikel Wettbewerb gestartet, wo wir unseren ersten Beitrag einreichten. Die Gewinner 
+erhalten ein Ticket zur RIPE89 in Prag. Dank hervorragender Arbeit der Verfasser und Rezensoren, hat unser Artikel
+gewonnen und wir sind sehr zufrieden, dass diese auch so gut in der Community ankommen.
+
+Danke an die fleißigen Mitschreiber, Leser und Rezenten, wir freuen uns darauf Euch auf der RIPE in Prag zu sehen!
```

Comment:
The last part is not related to the acknowledgements, isn't it? IMHO it should be a dedicated sentence.